book/02-git-basics/sections/viewing-history.asc

```diff
@@ -227,7 +228,7 @@ Those are only some simple output-formatting options to `git log` – there are
 ==== Limiting Log Output
 
 In addition to output-formatting options, `git log` takes a number of useful limiting options – that is, options that let you show only a subset of commits.
-You've seen one such option already – the `-2` option, which show only the last two commits.
+You've seen one such option already, the `-2` option, which shows only the last two commits.
```

Comment:
I'm new to this, I assume I can just reply...

I wouldn't put a colon there.  You could put a semi-colon there as a
connection between two related concepts on one sentence.

On Fri, Aug 5, 2016 at 4:00 PM, Ben Straub <notifications@github.com> wrote:

> In book/02-git-basics/sections/viewing-history.asc
> <https://github.com/progit/progit2/pull/630#discussion_r73765646>:
>
> > @@ -227,7 +228,7 @@ Those are only some simple output-formatting options to `git log` – there are
> >  ==== Limiting Log Output
> >
> >  In addition to output-formatting options, `git log` takes a number of useful limiting options – that is, options that let you show only a subset of commits.
> > -You've seen one such option already – the `-2` option, which show only the last two commits.
> > +You've seen one such option already, the `-2` option, which shows only the last two commits.
>
> This seems like a good place for a colon though, right? A comma doesn't
> quite feel right to me.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/progit/progit2/pull/630/files/2cb2f4600180163db5ac70f1f440e28cbf88c975#r73765646>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABmTbgooAmetzVUolJccv30rfErvABncks5qc7JjgaJpZM4JYCxK>
> .
>